test/conformance/parallel/ptptestconfig.yaml

```diff
@@ -0,0 +1,21 @@
+---
+global:
+ enablecontinuous: false
+masteroffsetcontinuous:
+ enable: true
+ failfast: true
+ minoffset: -100
+ maxoffset: 100
+ duration: 10
+testacontinuous:
+ enable: true
+ failfast: true
+ minoffset: -100
+ maxoffset: 100
+ duration: 10
+testbcontinuous:
```

Comment:
won't this be duplicate ?Since you only have masteroffsetcontinuous 
we should only have that and not duplicate tests

Reply:
There is a plan to put these variables as global inside the config.